lib/buckets/moray.js

```diff
@@ -816,7 +816,6 @@ Moray.prototype.ping = function ping(opts, cb) {
 
 Moray.prototype.search = function search(options) {
```

Comment:
I've removed this in the muskie/libmanta consolidation work I've started. I don't think we need to maintain the `search` functionality so this can just be removed.